src/main/java/seedu/address/storage/elderly/JsonSerializableElderly.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.storage.elderly;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.FriendlyLink;
+import seedu.address.model.ReadOnlyElderly;
+import seedu.address.model.person.Elderly;
+import seedu.address.storage.JsonAdaptedPerson;
+
+/**
+ * An Immutable Elderly that is serializable to JSON format.
+ */
+@JsonRootName(value = "elderly")
+public class JsonSerializableElderly {
```

Comment:
Perhaps we should change these to plural to to make it  clear that this class is a list of elderlies
```suggestion
@JsonRootName(value = "elderlies")
public class JsonSerializableElderlies {
```

We can change this in the next pull request